src/gui/dialog.hpp

```diff
@@ -81,6 +82,24 @@ class Dialog
     MenuManager::instance().set_dialog(std::move(dialog));
   }
 
+  // the callback is called regardless the state(enabled/disabled) of confirmation dialog
+  static void show_confirmation(const std::string& text, const std::function<void ()>& callback)
```

Comment:
This method should only render confirmation dialoges. It should not depend on g_config->confirmation_dialog. 

@tobbi  Do you think that adding a static method like this is ok?